tests/nemo_text_processing/fr/test_cardinal.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
```

Comment:
please use `Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` for this and the rest test_*.py files